model/src/main/kotlin/Project.kt

```diff
@@ -63,8 +63,8 @@ data class Project(
     val authors: SortedSet<String> = sortedSetOf(),
 
     /**
-     * The list of licenses the authors have declared for this project. This does not necessarily correspond to the
-     * licenses as detected by a scanner. Both need to be taken into account for any conclusions.
+     * The list of licenses declared for this project. This does not necessarily correspond to the licenses as detected
```

Comment:
Same as above, recommend to replace "declared for this project" to "declared in the metadata for this project"